google/resource-snippets/cloudkms-v1/alternatives/tf/cloudkms.tf

```diff
@@ -0,0 +1,45 @@
+variable "project_id" {}
+variable "deployment" {}
+
+provider "google" {
+  project = var.project_id
+  region  = "us-central1"
+  zone    = "us-central1-c"
+}
+
+locals{
+    user = "user:zyanshu@google.com"
+}
+
+resource "google_kms_key_ring" "default" {
+  name     = "test--keyring"
```

Comment:
This should be `test--keyRing` according to `test_alternates.sh`. 

Nit: Can we remove the two dashes? It seems odd and the name is really from a bug where `properties["time"]` resolves to empty in [deploymentmanager-samples/kms.jinja](https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/cloudkms-v1/kms.jinja#L20).